docs/root/development/performance/device_conditions.rst

```diff
@@ -30,7 +33,45 @@ Experimentation method
 Modified versions of the "hello world" example apps were used to run the following experiments.
 See the iOS/Android sections below for instructions on building the examples.
 
-Throughout all of the following steps, we tested to make sure that network requests succeeded
+iOS
```

Comment:
I feel like these are general observations with some actions. Could we instead write up very specific steps on how you were able to reproduce the experimentation.

Something like:
```
1. Start the python service
2. Switched between WiFi to cellular
3. Wait for logs to appear
...
``

This way a reader can precisely reproduce the experimentation steps and read more about the observations you noticed in the analysis

Reply:
Updated